catalog/transformer/catalog-transformer-streaming-impl/src/main/java/org/codice/ddf/transformer/xml/streaming/impl/GmlHandler.java

```diff
@@ -64,6 +69,30 @@
 
     private Stack<String> state;
 
+    private static Set<AttributeDescriptor> attributeDescriptors = new HashSet<>();
+
+    static {
+        attributeDescriptors.add(new AttributeDescriptorImpl(Metacard.GEOGRAPHY,
+                true /* indexed */,
+                true /* stored */,
+                false /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.GEO_TYPE));
+
+        attributeDescriptors.add(new AttributeDescriptorImpl(Validation.VALIDATION_WARNINGS,
```

Comment:
I think all of these attribute descriptors are defined in `catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/types/`.  You should use the already defined versions.

Here is where the validation attributes are defined.
https://github.com/codice/ddf/blob/76975e51a8093358329ff828afbb7ae82a1adba7/catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/types/ValidationAttributes.java#L37-L48

I think this location attribute descriptor is the same as the geography one above.
https://github.com/codice/ddf/blob/f8542dd272bd4950cbcd7dabea289709494fd622/catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/types/CoreAttributes.java#L78-L83